app/v1/models/request/Postcode.scala

```diff
@@ -17,27 +17,31 @@
 package v1.models.request
 
 import play.api.Logger
-import play.api.libs.json.{JsString, Reads, Writes, __}
-
-import scala.util.matching.Regex
+import play.api.libs.json.{JsPath, JsString, JsonValidationError, Reads, Writes, __}
```

Comment:
Can you remove `JsPath` from the imports please

Reply:
done